src/pages/Home/Home.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import CategoryPopover from '../../components/CategoryPopover';
+import LabelsPopover from '../../components/LabelsPopover/LabelsPopover';
```

Comment:
`LabelsPopover` has index file in the folder, so you don't need `/LabelsPopover/LabelsPopover` in the path.